notebooks/SMAP/smap_rasterio_issues.ipynb

```diff
@@ -0,0 +1,2339 @@
+{
```

Comment:
<p>I don't actually have this file in my smap_data directory. But I don't even see that directory in your branch: <a href="https://github.com/nsidc/NSIDC-Data-Tutorials/tree/CRYO-84/notebooks/SMAP" target="_blank">https://github.com/nsidc/NSIDC-Data-Tutorials/tree/CRYO-84/notebooks/SMAP</a> Maybe this is missing? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/91/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>